com.unity.multiplayer.mlapi/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -8,6 +6,7 @@
 
 namespace MLAPI.Metrics
 {
+#if MULTIPLAYER_TOOLS
```

Comment:
I don't know if this was intentionally removed, but the #endif at the bottom is still there so either both need to be removed or this can be added back in if this was unintentional.